geth/swarm_api_proxy.go

```diff
@@ -0,0 +1,241 @@
+package geth
```

Comment:
- please remove this file
- in future, consider asking for advise before introducing something as dubious as this